UefiTestingPkg/FunctionalSystemTests/SmmPagingProtectionsTest/Smm/SmmPagingProtectionsTestSmm.c

```diff
@@ -11,14 +11,20 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 // MS_CHANGE - Entire file created.
 
 #include <PiSmm.h>
-
+#include <Library/BaseLib.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/DebugLib.h>
+#include <Library/MemoryAllocationLib.h>
 #include <Library/SmmServicesTableLib.h>
 #include <Library/SmmMemLib.h>
-#include <Library/DebugLib.h>
-#include <Library/BaseMemoryLib.h>
+
+#include <Library/PlatformSmmProtectionsTestLib.h>
 
 #include <Protocol/SmmExceptionTestProtocol.h>
 
+#include <Register/Intel/ArchitecturalMsr.h>
```

Comment:
do you need these?

Reply:
Yes, used by the new test to validate that SmmEntryPoint is not writable.